Makefile

```diff
@@ -64,6 +64,10 @@ bench-da-capo:
 	PYTHONPATH=krun/ JAVA_HOME=${JAVA_HOME} ${PYTHON} extbench/rundacapo.py | tee dacapo.csv
 	bin/csv_to_krun_json dacapo.csv
 
+bench-octane:
+	${PYTHON} extbench/runoctane.py
+	bin/csv_to_krun_json octane.v8.results
+
 export-graphs:
 	${PYTHON} export_all_graphs.py
```

Comment:
Well, this is interesting. I wasn't aware that this line was in the Makefile, and I don't *think* we have an export_all_graphs script. Also, all the scripts have changed, so this should target should probably be deleted, in this PR or another one.